12_findTheOldest/solution/findTheOldest-solution.js

```diff
@@ -1,19 +1,17 @@
-const findTheOldest = function (array) {
-  return array.reduce((oldest, currentPerson) => {
-    const oldestAge = getAge(oldest.yearOfBirth, oldest.yearOfDeath);
-    const currentAge = getAge(
-      currentPerson.yearOfBirth,
-      currentPerson.yearOfDeath
-    );
-    return oldestAge < currentAge ? currentPerson : oldest;
-  });
+const getAge = function (person) {
+  // The nullish coalescing assignment operator
+  // evaluates to the right hand side if the left hand side is undefined
+  // Else it evaluates to the left hand side
```

Comment:
I'm not sure I'd explain it this way. Unlike `??` it doesn't just "evaluate to the right hand side", as it's not just an expression but rather an assignment statement.

I'd probably say something more like
```suggestion
  // only does the assignment if the left side evaluates to undefined or null
  // if the left side has any other value, no assignment happens
  // here, we use ??= to set the current year for our subtraction below only if there is no year of death
```

Reply:
Good point!